test/BlazorSignalR.Test.Client/Pages/ChatComponent.cs

```diff
@@ -86,7 +86,7 @@ public void DemoMethodList(DemoData[] data)
 
         private async Task<string> GetJwtToken(string userId)
         {
-            var httpResponse = await this._http.GetAsync($"/generatetoken?user={userId}");
+            var httpResponse = await this._http.GetAsync($"{_http.BaseAddress}generatetoken?user={userId}");
```

Comment:
Is the base address guaranteed to end with a '/'? If not, there's no harm in putting an extra one anyways to just be sure.

Reply:
I think so. I had there a / there before. The server does not route this correctly then. But I can insert a check and add a / conditionally.